### ISSUE TITLE
Separate the Screens : As there is an admin clerk

### ISSUE THREAD

**Issue by dakkiboy17** [rpfilomeno/hmr-main#32]
Approval of Auction on Private
Adding of Images on auction
Bidding history of an auction

**Comment by rpfilomeno** [comment]
not separate screen, based on role. actions shall not be allowed

**Comment by dakkiboy17** [comment]
Yes we need to have separate screens as the clerk cannlt see data of mai tenance.
